--- jigdo-0.7.1/configure.debug	2004-06-20 16:49:15.000000000 -0400
+++ jigdo-0.7.1/configure	2005-04-15 18:19:10.000000000 -0400
@@ -2723,8 +2723,6 @@
 ac_link='$CXX -o conftest$ac_exeext $CXXFLAGS $CPPFLAGS $LDFLAGS conftest.$ac_ext $LIBS >&5'
 ac_compiler_gnu=$ac_cv_cxx_compiler_gnu
 
-CFLAGS=`echo "$CFLAGS" | sed 's/\(^\| \)-g\( \|$\)/ /'`
-CXXFLAGS=`echo "$CXXFLAGS" | sed 's/\(^\| \)-g\( \|$\)/ /'`
 IF_GXX2="#"
 IF_NOGXX2=""
 if test "$GXX" = yes; then
