Index: src/cachefile.hh
===================================================================
RCS file: /cvsroot/jigdo/jigdo/src/cachefile.hh,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -b -B -r1.3 -r1.4
--- src/cachefile.hh	2 Jul 2005 22:05:03 -0000	1.3
+++ src/cachefile.hh	21 Jul 2005 11:31:43 -0000	1.4
@@ -1,4 +1,4 @@
-/* $Id: cachefile.hh,v 1.3 2005/07/02 22:05:03 atterer Exp $ -*- C++ -*-
+/* $Id: cachefile.hh,v 1.4 2005/07/21 11:31:43 atterer Exp $ -*- C++ -*-
   __   _
   |_) /|  Copyright (C) 2001-2002  |  richard@
   | \/¯|  Richard Atterer          |  atterer.net
@@ -94,7 +94,7 @@
       only valid until the next database operation. */
   Status findName(const byte*& resultData, size_t& resultSize,
                   const string& fileName,
-                  long long int& resultFileSize, time_t& resultMtime);
+                  off_t& resultFileSize, time_t& resultMtime);
 
   /** Insert/overwrite entry for the given file (name must be
       absolute, file must have the supplied mtime and size). The data
Index: src/cachefile.cc
===================================================================
RCS file: /cvsroot/jigdo/jigdo/src/cachefile.cc,v
retrieving revision 1.7
retrieving revision 1.8
diff -u -b -B -r1.7 -r1.8
--- src/cachefile.cc	2 Jul 2005 22:05:03 -0000	1.7
+++ src/cachefile.cc	21 Jul 2005 11:31:43 -0000	1.8
@@ -1,4 +1,4 @@
-/* $Id: cachefile.cc,v 1.7 2005/07/02 22:05:03 atterer Exp $ -*- C++ -*-
+/* $Id: cachefile.cc,v 1.8 2005/07/21 11:31:43 atterer Exp $ -*- C++ -*-
   __   _
   |_) /|  Copyright (C) 2001-2003  |  richard@
   | \/¯|  Richard Atterer          |  atterer.net
@@ -131,7 +131,7 @@
 //________________________________________
 
 Status CacheFile::findName(const byte*& resultData, size_t& resultSize,
-    const string& fileName, long long int& resultFileSize,
+    const string& fileName, off_t& resultFileSize,
     time_t& resultMtime) {
   DBT key; memset(&key, 0, sizeof(DBT));
   key.data = const_cast<char*>(fileName.c_str());
